autogen/browser_utils/mdconvert.py

```diff
@@ -0,0 +1,1046 @@
+# ruff: noqa: E722
```

Comment:
I feel this code is super helpful. Thinking should we rearrange these classes into other folders or not... 

Many of these features can be great for RAG, Devil, etc...